### PR TITLE
Add Attachments

### PR DESCRIPTION
**Pull request by SonofGod1998**
This commit has Byakuren, Doremy, Futo, Ichirin, Joon and Kasen added their attachments

### CODE REVIEW

File: character-sql/byakuren.sql [hunk 1]
```diff
@@ -227,6 +240,8 @@ INSERT INTO `moves` (
 			"active": "8f",
 			"recovery": "27f",
 			"damage": "825"
```
Comment (JustAPenguin9) [inline character-sql/byakuren.sql:242]: Missing a `,` at the end of the line.

**Review by JustAPenguin9 — CHANGES_REQUESTED** [review]
doremy, futo, ichirin, joon, kasen, sql files verified. Only byakuren.sql had an issue.

**Comment by SonofGod1998** [comment]
I swear I'll finish Nitori's attachments, but please fix Miko's Red Cape 🙏

**Comment by SonofGod1998** [comment]
I have FINALLY added Reimu, finished up Nitori and did everything that was on the wiki for Mokou. I should've gone to bed .-.

**Review by JustAPenguin9 — APPROVED** [review]

**Comment by JustAPenguin9** [comment]
The remaining characters are filed under issue #103

**Pull request merged**